Connector Listing (#474)
ConduitIO/conduit: docs/connectors.md

# Conduit Connectors

> Connectors are an integral part of Conduit. Conduit ships with a couple of connectors that are built into the service to help developers bootstrap pipelines much more quickly. The built-in connectors include Postgres, File, Kafka and Amazon S3 connectors.

Reviewer: Here it mentions postgres as a builtin connector, but in the Readme it doesn't. It also doesn't mention the generator connector here.

Author: fixed

The difference between Conduit connectors and those you might find from other services is that Conduit connectors are Change Data Capture-first (CDC). CDC allows your pipeline to only get the changes that have happened over time instead of pulling down an entire upstream data store and then tracking diffs between some period of time. This is critical for building real-time event-driven pipelines and applications. But, we'll note where connectors don't have CDC capabilities.

## Roadmap & Feedback

> Based on what the Conduit team hears from developers, a [roadmap]() has been built to facilitate the connectors that need to be built for Conduit. If you have a connector that you really want to see supported or lend your vote for a particular connector, check out that roadmap and upvote the associated issue!

Reviewer 2: Nitpick: link for roadmap missing.

Author: fixed

Author: argh.

Reviewer 2: I've approved, feel free to merge when you push this change too.

Author: fixed.

## Connectors

### Support Types

* `Conduit` - These are connectors that are built by the Conduit. Any issues or problems filed on those repos will be respond to by the Conduit team.
* `Community` - A community connector is one where a developer created a connector and they're the ones supporting it not the Conduit team.
* `Legacy` - Some connectors are built using non-preferred methods. For example, Kafka Connect connectors can be used on Conduit. This is considered a stop gap measure until `conduit` or `community` connectors are built.

At this time, Conduit does not have any commerically supported connectors.

### Source & Destination

> Source means, does the connector have the ability to get data from the upstream data store. Destination means, does the connector have the ability to write to the downstream data store.

Reviewer: ```suggestion
Source means that the connector has the ability to get data from the upstream data store. Destination means that the connector has the ability to write to the downstream data store.
```

Author: fixed

### The List

> | Connector | Source | Destination | Support |

Reviewer 2: Did we intentionally leave out the Kafka Connect wrapper from the list?

> |-----------|-------|----|-------------|

Reviewer: IMO it would be better to push all the builtin connectors to the top of the list

Author: I'm leaving the list alphabetical to make it easier to search.

# Conduit Connectors

Connectors are an integral part of Conduit. Conduit ships with a couple of connectors that are built into the service to help developers bootstrap pipelines much more quickly. The built-in connectors include Postgres, File, Random Data Generator, Kafka and Amazon S3 connectors.

> The difference between Conduit connectors and those you might find from other services is that Conduit connectors are Change Data Capture-first (CDC). CDC allows your pipeline to only get the changes that have happened over time instead of pulling down an entire upstream data store and then tracking diffs between some period of time. This is critical for building real-time event-driven pipelines and applications. But, we'll note where connectors don't have CDC capabilities.

Reviewer 2: > But, we'll note where connectors don't have CDC capabilities.

In the connector repos, not here, right?

Author: For now, in the connector repos.
